src/components/NamesInAuctionTableCondensed.vue

```diff
@@ -62,6 +82,7 @@ import AppLink from '@/components/AppLink'
 import { formatAePrice } from '@/utils/format'
 import ValueHashEllipsed from '@/components/ValueHashEllipsed'
 import DatetimeLabel from '@/components/DatetimeLabel'
+import { namesHints } from '~/utils/hints/namesHints'
```

Comment:
Is there any reason why this import is different? `~` vs `@`

Reply:
Nope, it's an issue of my IDE. I will try to tune it. Fixed